src/app/admin/components/user-privileges/privileges-list/privileges-list.component.ts

```diff
@@ -0,0 +1,91 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { first } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+import { trigger, transition, style, animate, query, stagger } from '@angular/animations';
+
+@Component({
+  selector: 'clark-privileges-list',
+  templateUrl: './privileges-list.component.html',
+  styleUrls: ['./privileges-list.component.scss'],
+  animations: [
```

Comment:
I think as animations grow we should identify a cutoff point where we place them in their own file. I don't think this one here is necessarily too big, but might want to be a guideline we establish

Reply:
I definitely agree with this.

Reply:
I was noodling on this at lunch, I'm thinking it might be a good idea to decree that all animations should _always_ live in a separate file. A cutoff is a bit of a slippery slope I feel.

Reply:
I like it 👍

Reply:
Collection-dashboard animations are extracted to their own files